[2024-01-31 13:50:35,287] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_5.create_agg_table manual__2024-01-31T13:50:27.684979+00:00 [queued]>
[2024-01-31 13:50:35,316] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_5.create_agg_table manual__2024-01-31T13:50:27.684979+00:00 [queued]>
[2024-01-31 13:50:35,316] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 13:50:35,317] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-01-31 13:50:35,317] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-01-31 13:50:35,368] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_agg_table> on 2024-01-31 13:50:27.684979+00:00
[2024-01-31 13:50:35,389] {standard_task_runner.py:52} INFO - Started process 4318 to run task
[2024-01-31 13:50:35,410] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_5', 'create_agg_table', 'manual__2024-01-31T13:50:27.684979+00:00', '--job-id', '87', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp8qs5mkx4', '--error-file', '/tmp/tmp3jel7zze']
[2024-01-31 13:50:35,415] {standard_task_runner.py:80} INFO - Job 87: Subtask create_agg_table
[2024-01-31 13:50:35,873] {task_command.py:369} INFO - Running <TaskInstance: project_6_5.create_agg_table manual__2024-01-31T13:50:27.684979+00:00 [running]> on host 8ebb6ccf62d7
[2024-01-31 13:50:36,690] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_5
AIRFLOW_CTX_TASK_ID=create_agg_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-31T13:50:27.684979+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-31T13:50:27.684979+00:00
[2024-01-31 13:50:36,848] {base.py:68} INFO - Using connection ID 'postgres_dwh' for task execution.
[2024-01-31 13:50:36,973] {dbapi.py:208} INFO - Running statement: 
                create table if not exists test_1 (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
                )
        , parameters: None
[2024-01-31 13:50:36,998] {postgres.py:94} INFO - NOTICE:  relation "test_1" already exists, skipping

[2024-01-31 13:50:37,238] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_5, task_id=create_agg_table, execution_date=20240131T135027, start_date=20240131T135035, end_date=20240131T135037
[2024-01-31 13:50:37,409] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-31 13:50:37,936] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
